bot/exts/moderation/verification.py

```diff
@@ -519,12 +539,32 @@ async def on_member_join(self, member: discord.Member) -> None:
         if member.guild.id != constants.Guild.id:
             return  # Only listen for PyDis events
 
+        raw_member = await self.bot.http.get_member(member.guild.id, member.id)
+
+        # Only send the message to users going through our gating system
+        if raw_member["is_pending"]:
+            await self.member_gating_cache.set(raw_member.id, True)
```

Comment:
This should be `member.id`

```
2020-10-05 16:42:18 | bot | ERROR | Unhandled exception in on_member_join.
Traceback (most recent call last):
  File "/home/mark/repos/python/bot-pydis/.venv/lib/python3.8/site-packages/discord/client.py", line 333, in _run_event
    await coro(*args, **kwargs)
  File "/home/mark/repos/python/bot-pydis/bot/exts/moderation/verification.py", line 546, in on_member_join
    await self.member_gating_cache.set(raw_member.id, True)
AttributeError: 'dict' object has no attribute 'id'
```

Reply:
Or `raw_member["id"]` I suppose.

Reply:
Addressed in 082e26342eb3faf104523334cdb87e07eda03db3